### ISSUE TITLE
Allow team names with spaces

### ISSUE THREAD

**Issue by jmptbl** [go-gitea/gitea#16140]
- Gitea version (or commit ref): 1.4.0
- Git version:N/A
- Operating system:Docker
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [X] MySQL
  - [ ] MSSQL
  - [ ] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [ ] No
- Log gist:

## Description
Is it possible to allow team names with spaces in them? Github allows this, so having it in Gitea would help me with mirroring our Github organisation to our Gitea instance. Can't imagine why spaces aren't allowed...
...

## Screenshots
![Screenshot from 2021-06-13 00-47-49](https://user-images.githubusercontent.com/1299110/121790569-111b1600-cbe1-11eb-91c3-99287b86ddf2.png)

**Comment by nobodyinperson** [comment]
Probably related (there seems to be something quite broken with team names): if you put dashes in a team name (instead of spaces, the logical choice), the team is auto-completed when composing an issue, but upon preview/send it ends up as plaintext and no notifications are sent out.

![grafik](https://github.com/go-gitea/gitea/assets/19148271/c3f13c2c-e41b-44a5-9b43-49788120e8b3)

**Comment by wxiaoguang** [comment]
Yep, not really feasible (as https://github.com/go-gitea/gitea/issues/16140#issuecomment-2088170594 said)

You can add descriptions for teams
